source/code/providers/SCX_FileSystemStatisticalInformation_Class_Provider.cpp

```diff
@@ -98,16 +99,16 @@ static void EnumerateOneInstance(
             inst.AverageTransferTime_value(ddata1);
         }
 
-        if (diskinst->GetDiskSize(data1, data2))
+        if (diskinst->GetDiskSize(data1, data2, data3))
         {
             inst.FreeMegabytes_value(data2);
             inst.UsedMegabytes_value(data1);
             unsigned char freeSpace = 100;
             unsigned char usedSpace = 0;
             if (0 < data1+data2)
             {
-                freeSpace = (unsigned char) SCXCoreLib::GetPercentage(0, data2, 0, data1+data2);
-                usedSpace = (unsigned char) SCXCoreLib::GetPercentage(0, data2, 0, data1+data2, true);
+                freeSpace = (unsigned char) SCXCoreLib::GetPercentage(0, data2, 0, data3);
+                usedSpace = (unsigned char) SCXCoreLib::GetPercentage(0, data1, 0, data3);
```

Comment:
line#111: previous GetPercentage's 5th parameter value is true, maybe we should keep it?
https://github.com/microsoft/pal/blob/06c611367b1c31b3b6a6b65a9f58117fe54a9e95/source/code/scxcorelib/util/scxmath.cpp#L56

Reply:
No, Keeping 5th parameter as true would result into 100-data percentage calculation which we don't need

Reply:
ok